app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -29,16 +45,35 @@ class ArNavigationActivity : AppCompatActivity() {
         }
     }
 
-    private val mapboxNavigation by lazy {
-        MapboxNavigation(this, Mapbox.getAccessToken()!!)
+    private val arLocationEngine by lazy {
```

Comment:
Why do we need `locationEngine` here?

Reply:
We need it to make Navigation work properly. In opposite way I will create default location provider which does not work well and we will miss offRoad event